test/types/objectSearch.test.ts

```diff
@@ -127,4 +127,27 @@ describe('objectSearch', () => {
     expect(tester.text()).not.toContain(results[1].name);
     expect(tester.text()).toContain(results[2].name);
   });
+
+  [true, false].forEach((searchOnEmpty: boolean) => {
+    it(`Works for searchOnEmpty={${searchOnEmpty}} prop`, async () => {
+      const { field, searchTerm, results, props } = getComponentDefaults({})
+        , tester = (await new Tester(ObjectSearch, { props: { ...props, searchOnEmpty }}).mount() as any)
+        , callsOnFocus: number = searchOnEmpty ? 1 : 0
```

Comment:
This is clear, but you could use `+searchOnEmtpy` instead of a ternary

Reply:
You are a bad person.